R/zchunk_L222.en_transformation.R

```diff
@@ -60,9 +60,15 @@ module_energy_L222.en_transformation_DISABLED <- function(command, ...) {
 
     all_data <- list(...)[[1]]
 
+    # Silencing global variable package checks
+    GCAM_region_ID <- calOutputValue <- coefficient <- fuel <- half.life <- input.cost <- lifetime <-
+    median.shutdown.point <- minicam.energy.input <- minicam.non.energy.input <- object <-
+    profit.shutdown.steepness <- region <- remove.fraction <- sector <- sector.name <- share.weight <-
+    shutdown.rate <- steepness <- stub.technology <- subsector <- subsector.name <- supplysector <-
+    technology <- to.value <- value <- year <- year.fillout <- year.x <- year.y <- NULL
+
     # Load required inputs
     GCAM_region_names <- get_data(all_data, "common/GCAM_region_names")
-    fuel_energy_input <- get_data(all_data, "energy/fuel_energy_input")
```

Comment:
Was this not used?

Reply:
it was not.

Reply:
Does another chunk use the fuel_energy_input.csv?  I notice it is still in the files listed in the chunk